src/main/java/io/apimatic/core/logger/SdkLogger.java

```diff
@@ -58,22 +58,20 @@ public SdkLogger(final LoggingConfiguration config) {
     public void logRequest(Request request) {
         Level level = config.getLevel() != null ? config.getLevel() : Level.INFO;
 
-        String url = request.getUrl();
         String queryParameter = CoreHelper.getQueryParametersFromUrl(request.getQueryUrl());
         String contentType = request.getHeaders().value(LoggerConstants.CONTENT_TYPE) != null
                 ? request.getHeaders().value(LoggerConstants.CONTENT_TYPE)
                 : "";
 
         Map<String, Object> requestArguments = new LinkedHashMap<String, Object>();
         requestArguments.put(LoggerConstants.METHOD, request.getHttpMethod());
-        requestArguments.put(LoggerConstants.URL, url);
-        requestArguments.put(LoggerConstants.CONTENT_TYPE, contentType);
+        requestArguments.put(LoggerConstants.URL, request.getUrl());
```

Comment:
Nits, Non-blocker: We can minimize the number of lines by putting the decision on queryParams in Path.
```Java
requestArguments.put(LoggerConstants.URL, getReqestUrl(request));

// following is the method implementation
private string getRequestUrl(Request request) {
    if (requestLogOptions.shouldIncludeQueryInPath()) {
        return request.getQueryUrl();
    }

    return request.getUrl();
}
```

Reply:
Resolved by extracting the logic inside a private method